internal/worker/task.go

```diff
@@ -74,6 +75,9 @@ func (worker *Worker) runTask(
 			ServerSecret: agentAwareTask.ServerSecret,
 			ClientSecret: agentAwareTask.ClientSecret,
 			TaskID:       agentAwareTask.TaskId,
+			AdditionalEnvironment: map[string]string{
+				"CIRRUS_SENTRY_TAGS": fmt.Sprintf("cirrus.task_id=%d", agentAwareTask.TaskId),
```

Comment:
What do you think of including `upstream` worker name and possibly host of the RPC endpoint for easier aggregation on Setry side?

Reply:
See https://github.com/cirruslabs/cirrus-cli/pull/589/commits/b2e113420ed9c777092d27e4c7da37fb752e5dc4.